bundles/specmate-emfrest-api/src/com/specmate/emfrest/api/RestServiceBase.java

```diff
@@ -99,6 +99,11 @@ public Object delete(Object object) throws SpecmateException, SpecmateValidation
 		return null;
 	}
 
+	@Override
+	public boolean isStatusService() {
+		return false;
+	}
+

```

Comment:
Hm, wouldn't it be more elegant if StatusRestService would implement IStatusService, instead of using composition? Because then you would not need to have this isStatusService() method but can check via instanceof if something is a IStatusService (which is exported in the API).

Reply:
done